app/views/spree/layouts/_admin_body.html.haml

```diff
@@ -59,6 +59,8 @@
                 %span= yield :sidebar_title
             = yield :sidebar
 
+  = render partial: "admin/terms_of_service_banner"
```

Comment:
I prefer to have the condition here instead of adding indent to the partial.
```suggestion
  = render partial: "admin/terms_of_service_banner" if @terms_of_service_banner
```

Or, we use a helper and maybe Rails already knows that it's a partial?
```suggestion
  = render "admin/terms_of_service_banner" if tos_need_accepting?
```

Reply:
It's clearer indeed :)